app/domain/person/household.rb

```diff
@@ -40,24 +40,24 @@ def assign
     end
   end
 
-  def persist!
+  def persist!(current_user)
+    @current_user = current_user
```

Comment:
Kannst du den @current_user noch in den initializer verschieben und einen attr_reader dafür erstellen?